### PR TITLE
routing is implemented

### PR DESCRIPTION
**Pull request by harmmselution**

### CODE REVIEW

File: src/components/Categories/Categories.tsx
Comment (Segacnd) [inline src/components/Categories/Categories.tsx:1]: seems that need add type for component

Comment (harmmselution) [reply src/components/Categories/Categories.tsx:1]: sosi

File: src/components/TodosContainer/TodosContainer.tsx
Comment (Segacnd) [inline src/components/TodosContainer/TodosContainer.tsx:1]: please, make a base dirname, like in all directories.
also u forget about component type))

**Review by Segacnd — APPROVED** [review]
good job

**Pull request merged**